tests/cli/test_sb.py

```diff
@@ -83,6 +84,20 @@ def test_sb_run_nonexist_host_file(self):
         result = self.cmd('sb run --host-file ./nonexist.yaml', expect_failure=True)
         self.assertEqual(result.exit_code, 1)
 
+    def test_sb_benchmark_list(self):
+        """Test sb benchmark list."""
+        self.cmd('sb benchmark list', checks=[JMESPathCheck('length(@)', len(BenchmarkRegistry.benchmarks))])
+
+    def test_sb_benchmark_list_nonexist(self):
+        """Test sb benchmark list, give a non-exist benchmark name, should fail."""
+        result = self.cmd('sb benchmark list -n non-exist-name', expect_failure=True)
+        self.assertEqual(result.exit_code, 1)
+
+    def test_sb_benchmark_list_parameters(self):
+        """Test sb benchmark list-parameters."""
+        self.cmd('sb benchmark list-parameters', checks=[NoneCheck()])
+        self.cmd('sb benchmark list-parameters -n pytorch-[a-z]+', checks=[NoneCheck()])
```

Comment:
Do we need to check the exit_code?

Reply:
by default `expect_failure` is `False` so exit code is expected to be 0, otherwise the case will fail